tests/plugins/test_add_image_content_manifest.py

```diff
@@ -112,11 +116,11 @@
 )
 
 REMOTE_SOURCES = [{
-    'build_args': None,
-    'configs': None,
-    'request_id': CACHITO_REQUEST_ID,
-    'url': None,
-    'name': None,
+    'id': CACHITO_REQUEST_ID,
+    # 'url': 'some url',
```

Comment:
these can be removed I'd say, since they aren't used by that plugin, or you can just leave them here for consistency

Reply:
yeah, the idea was to use only `id` but keep the others there as comments to make it look consistent with the plugin result